subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/maven/MavenLocalRepoResolveIntegrationTest.groovy

```diff
@@ -133,32 +133,6 @@ class MavenLocalRepoResolveIntegrationTest extends AbstractDependencyResolutionT
         failure.assertHasCause("Unable to parse local Maven settings: " + m2.userSettingsFile.absolutePath)
     }
 
-    def "fail if metadata in local repo corrupts and remote repo is #remoteStatus"() {
-        given:
-        def corruptLocalPom = m2.mavenRepo().rootDir.file('group/projectA/1.2/projectA-1.2.pom')
-        corruptLocalPom << "invalid content"
-
-        if (remoteStatus == 'valid') {
-            def remoteRepo = maven("remote")
-            remoteRepo.module('group', 'projectA', '1.2').publish()
-
-            buildFile << """
-            repositories{
-                maven { url "${remoteRepo.uri}" }
-            }
-            """
-        }
-
-        when:
-        runAndFail 'retrieve'
-
-        then:
-        failure.assertHasCause("Could not parse POM $corruptLocalPom.absolutePath")
-
-        where:
-        remoteStatus << ['valid', 'invalid']
-    }
-
```

Comment:
I don't quite understand why this is removed, could you please explain slightly?

Reply:
It was failing and I saw that you removed it in your previous PR. So I copied you :).

Reply:
Phew, I was just reverting it to a previous version. Never mind.